src/components/Posts/ComposePost.tsx

```diff
@@ -79,19 +108,44 @@ const ComposePost = ({ placeholder, minRows }: ComposePostProps) => {
               <IconButton size="small">
                 <AddPhotoAlternateOutlinedIcon />
               </IconButton>
-              <IconButton size="small">
-                <EmojiEmotionsOutlinedIcon />
-              </IconButton>
+              <Stack direction="column">
+                <IconButton
+                  id="emoji-button"
+                  size="small"
+                  onClick={() => toggleEmojiPicker(false)}
+                >
```

Comment:
I feel like updating the setState inline should suffice, you just need to toggle using the `!` to get the same functionality.

```suggestion
                  onClick={() => setShowEmojiPicker(!showEmojiPicker)}
```

Reply:
The state is now set inline.